src/Sarif.Validation/Resources.resx

```diff
@@ -0,0 +1,138 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
```

Comment:
We moved the resource strings containing the rule descriptor descriptions and error messages from the SarifValidator project to the Sarif.Validation project.